File: GO/src/golang/01-Introduccion/funciones/funciones.go
package main

import "fmt"

// Función que recibe paámetros
func saludar(nombre string) {
	// fmt.Println("Hola Mundo")
	fmt.Println("Hola", nombre)
}

// Función qu retorna valores:
// Es importante que se debe indicar el tipo de dato que se va a retornar al ejecutarse la función
func sumar(a, b int) int {
	return a + b
}

// Dentro de la función principal debemos invocar la función para que se ejecute
func main() {
	saludar("Joel")
	r := sumar(5, 4)              // Esta ejecución los retorna pero no los imprime, se asigna a una variable para imprimirlo
	fmt.Println("La suma es:", r) // Esta función si los imprime
}

File: PYTHON/04-FUNCIONES/funciones.py

def saludar():
	global nombre
	nombre = 'Joel Palacio'
	print('Hola', nombre ,'desde la función saludar')
	return 'Hola Mundo desde la función saludar' # Retonra un valor e indica el término de la función



valor = saludar() # Llamada de la función para que se ejecute
print(valor)
print('Hola', nombre ,'Desde fuera de la función saludar, asignando el nombre de la variable "nombre" de manera global')


def saludo():
	nombre = 'JOEL'
	edad = 33
	return f'Hola {nombre}, tienes {edad} años de edad.'

saludo = saludo()
print(saludo)
